src/Model/Base/Project.php

```diff
@@ -15,6 +15,7 @@ class Project extends Model
     const TYPE_GOGS         = 'gogs';
     const TYPE_HG           = 'hg';
     const TYPE_BITBUCKET_HG = 'bitbucket-hg';
+    const TYPE_BITBUCKET_SV = 'bitbucket-server';
```

Comment:
@fejal Could you rename const `TYPE_BITBUCKET_SV` to `TYPE_BITBUCKET_SERVER` please?

Reply:
Yes, will do.